topydo/lib/ChangeSet.py

```diff
@@ -158,7 +163,18 @@ def _trim(self):
         for changeset in index[backup_limit:]:
             self.delete(changeset[0], p_write=False)
 
-    def get_backup(self, p_todolist):
+    def _get_backup(self):
```

Comment:
My suggestion is to keep `_get_backup` public, and accept an optional parameter for the timestamp. This eliminates the need for the two `get_backup_*` methods below, plus the need to keep `self.timestamp` around. If no timestamp is given, use the latest one (as in line 185 below).